packages/server/src/health/health.controller.ts

```diff
@@ -1,9 +1,23 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, HttpStatus, Res } from '@nestjs/common';
+import { Response } from 'express';
+import { HealthService } from './health.service';
 
 @Controller('health')
 export class HealthController {
-  @Get()
-  sendMessage(): string {
-    return 'Ok';
+  constructor(private readonly healthService: HealthService) {}
+
+  @Get('live')
+  healthLive(@Res() res: Response): Response<void> {
+    return res.status(HttpStatus.NO_CONTENT).send();
+  }
+
+  @Get('ready')
+  async healthReady(@Res() res: Response): Promise<Response<void>> {
+    const dbConnection = this.healthService.isDbReady();
+    if (!dbConnection) {
+      return res.status(HttpStatus.NOT_FOUND).send();
```

Comment:
Probably don't want to return a `NOT_FOUND` here since that implies at the resource/end point is not found.
[SERVICE_UNAVAILABLE](http://nestjs-doc.exceptionfound.com/enums/httpstatus.html#service_unavailable) seems like it would be a better fit since that is used for when a server has a resource that is not ready.

(400 type errors are perceived as an error caused by the client while 500 type errors represent something on the server side)

Reply:
fixed